### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by krisdo**
--nav bar sticks

TODO
--layout other microservices
--make nav bar appear after scrolling past photos service

### CODE REVIEW

File: client/src/components/Sticky.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  const element = useRef(null)
+
+  const handleScroll = () => {
+    window.scrollY > element.current.getBoundingClientRect().bottom
+      ? setSticky(true)
+      : setSticky(false)
+  }
+
+  // This function handle the scroll performance issue
+  const debounce = (func, wait = 20, immediate = true) => {
```
Comment (cmonez) [inline client/src/components/Sticky.jsx:14]: woahhh, explain this to me???

File: client/src/components/Sticky.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        if (!immediate) func.apply(context, args)
+      }
+      const callNow = immediate && !timeOut
+      clearTimeout(timeOut)
+      timeOut = setTimeout(later, wait)
+      if (callNow) func.apply(context, args)
+    }
+  }
+
+  useEffect(() => {
```
Comment (cmonez) [inline client/src/components/Sticky.jsx:30]: oooh using hooks i see!

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,12 @@
+FROM node:10
+
+WORKDIR /usr/src/app
+
+# why the *
```
Comment (cmonez) [inline Dockerfile:5]: why the.....

File: client/src/components/Sticky.jsx (hunk 1 above)
Comment (krisdo) [reply client/src/components/Sticky.jsx:14]: oops, I didn't use this. It's for scrolling so it doesn't jump around. I copied it but haven't used it.

File: Dockerfile (hunk 3 above)
Comment (krisdo) [reply Dockerfile:5]: oh old notes to ask during OH haha

**Pull request merged**